src/pages/StocktakeEditPage.js

```diff
@@ -178,18 +178,18 @@ export const StocktakeEditPage = ({
       />
     );
 
-    return <View style={newPageTopRightSectionContainer}>{program ? null : ManageStocktake}</View>;
+    return <View style={pageTopRightSectionContainer}>{program ? null : ManageStocktake}</View>;
```

Comment:
Not entirely convinced of this myself, but going to suggest it anyway:

```
const ManageStocktakeOrNull = program ? null : ManageStocktake;
return (
  <View style={...}>
    <ManageStocktakeOrNull/>
  </View>;
```

Reply:
same as above